server/forms.py

```diff
@@ -327,6 +327,12 @@ class CourseUpdateForm(BaseForm):
     active = BooleanField('Activate Course', default=True)
     timezone = SelectField('Course Timezone', choices=[(t, t) for t in pytz.common_timezones])
 
+class GradeVisibilityUpdateForm(BaseForm):
+    publish_tags = [tag.title() for tag in GRADE_TAGS if tag != 'private']
+    grades = SelectField('Publish Grade', choices=[(t, t) for t in publish_tags], default='Total')
```

Comment:
`grades` seemed somewhat confusing later on when it gets used. `tag` could be better. 

A better description might be along the lines
`Publish Scores With Tag:`